cnf-certification-test/operator/suite.go

```diff
@@ -104,23 +108,28 @@ func testOperatorInstallationWithoutPrivileges(env *provider.TestEnvironment) {
 		}
```

Comment:
add the compliant/non compliant object in the loop and if :
if n := len(permission.Rules[ruleIndex].ResourceNames); n > 0 
and add a rules and permission field to the operator